mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2370,6 +2370,26 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+{{ if ne $.TargetVersionName `ga` -}}
+			"pod_autoscaling": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: `The config for pod autoscaling.`,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"hpa_profile": {
```

Comment:
Would it make sense to add validation to make sure this only contains one of the valid enum values? (Generally that's good to do unless the list of valid values is expected to change in the future.)

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/validation#StringInSlice would be the func to use.